learning-bitmark/working-with-bitmarks/payment-for-bitmark-cli.md

```diff
@@ -0,0 +1,261 @@
+# Bitmark transaction payment
+
+In the Bitmark Property System, for each asset, each Bitmark Account is able to issue a bitmark for free, and has to pay for all the other transactions such as bitmark transfers or other bitmark issuances.
+
+The Bitmark CLI submits transactions to its connected node via RPC. Once receiving these transactions, the node processes them and returned the corresponding payment ids along with the payment amounts and addresses. Only when the payment - either by bitcoin or litecoin - is executed, a transaction is updated from `Pending` to `Verified` and is ready to be mined. The payment transaction needs to send:
```

Comment:
I find "to its connected node" confusing. Presumably you're talking about "the connected Bitmark server" or something like that.

Reply:
My intent here is just general comments, not (yet) line editing, but I wanted to offer a suggested update for this:

"The Bitmark CLI submits transactions to its connected Bitmark server via RPC. Once the server receives these transactions, it processes them and returns corresponding payment ids,  amounts, and addresses. The transaction is only updated from `Pending` to `Verified`, making it ready to be mined, once a payment is executed, either in bitcoin or litecoin.

In order for bitcoin or litecoin payment to be correctly recognized, freeing up the Bitmark transaction, it must include:"

Generally, I tried to clarify everything there, and to be more consistent about voice and verb tense.

Reply:
@shannona: I don't want to user the Bitmark server because it isn't and it creates the feeling of a centralized server control by Bitmark team. Bitmark-CLI doesn't connect to a server but a Bitmark node.

Reply:
Update the text